Articles/Blog/2022-03-sc21-rse-workshop.md

```diff
@@ -0,0 +1,204 @@
+# A Report on the SC21 Research Software Engineers in HPC (RSE-HPC-2021) Workshop
+
+**Hero Image:**
+
+- <img src='https://github.com/betterscientificsoftware/bssw.io/raw/master/images/Blog_2112_SC21.png' />
+
+#### Contributed by: [Charles Ferenbaugh](https://github.com/cferenba), [Sandra Gesing](https://github.com/sandragesing), [Simon Hettrick](https://github.com/SimonHettrick), and [Daniel S. Katz](https://github.com/danielskatz)
+#### Publication date: March 14, 2022
+
+The [RSE-HPC-2021 workshop](https://us-rse.org/rse-hpc-2021/) was held in
+November as part of the [SC21](https://sc21.supercomputing.org/) conference.
+Following up on the successful SC20 workshop, this year's full-day workshop
+included 20 speakers (with slides posted on the [workshop
+website](https://us-rse.org/rse-hpc-2021/agenda/)) and over 50 participants,
+in a hybrid format of onsite plus virtual content. The workshop's structure
+was designed to maximize audience participation, and consisted of six
+sessions, summarized below.
+
+### Conversation: Exascale computing and RSEs
+
+Lois Curfman McInnes of Argonne National Lab opened the workshop with a
+short talk and extended discussion on the subject "Does exascale computing
+change research software?" She described how software has become the
+foundation of sustained collaboration in many HPC domains, including
+computational science and engineering, data science, machine learning, and
+computational infrastructure. Many efforts in recent years have helped to
+improve scientific software, and the Exascale Computing Project (ECP) in
+particular has driven significant advances. Exascale computing has magnified
+the complexity and scope of traditional software challenges: adding support
+for heterogeneous architectures and next-generation science, and
+coordinating a wide range of software and teams, with all the resulting
+technical and sociological challenges. In response, ECP has developed an
+open, hierarchical software ecosystem, building a "team of teams" based on
+community collaboration and shared policies, while advancing scientific
+productivity through better scientific software. She concluded her talk with
+a shout-out to her ECP RSE colleagues at Argonne and elsewhere, describing
+them as "the linchpin of all things exascale - and beyond"!
+
+### Short talks
+
+Marion Weinzierl gave the first short talk on "Training HPC RSEs: What we
+learned from our performance." This was based on training workshops that
+they gave, with seven sessions over seven months. These lessons were
```

Comment:
Maybe I missed this, but who is "they" in this sentence? Maybe clairfy?

Reply:
Modified text to address this